index.js

```diff
@@ -91,7 +91,8 @@ function serveStatic(root, options) {
       path = ''
     }
 
-    // create send stream
+    // create send stream (etag computation and other options are taken care of
```

Comment:
> If someone thinks this comment is missing something, then they can, in turn, add some more there about their own need. As an iterative process. 

Correct, but we're iterating here, on the PR.

> But let us not waste our energies on this very unimportant PR: would you accept a PR with a comment explicitly mentioning etag and (I would try my best) not overly specific?

No, as you can see, I would not accept that. If the purpose is to point them to find `send` functionality for people who are not simply following the code or using a debugger, then we need to provide _all_ the information needed to do so. I don't have enough time to plan OR wack-a-mole unless you are willing to pay me.